### ISSUE TITLE
CUDA installation missing in GCP code

### ISSUE THREAD

**Issue by harishanand95** [Open-UAV/openuav-turbovnc#25]
1. nvidia-cuda-toolkit

**Comment by harishanand95** [comment]
Tesla K80 CUDA 10.2 download link [https://us.download.nvidia.com/tesla/440.118.02/NVIDIA-Linux-x86_64-440.118.02.run](https://us.download.nvidia.com/tesla/440.118.02/NVIDIA-Linux-x86_64-440.118.02.run)

**Comment by harishanand95** [comment]
Add all existing users to docker. 
```
for dir in /home/*/ 
do
    dir=${dir%*/}
    sudo usermod -a -G docker ${dir##*/}
done

```

**Comment by harishanand95** [comment]
Fixed in 62dad998f66f7b398dd2d3e550c82769ce762349 , abd33326f12af0b56f2b52ef1377f34656894827, b430309700fb97161004257f42b4ae5974be2922
